pkg/loki/modules.go

```diff
@@ -438,6 +438,10 @@ func (t *Loki) initRuler() (_ services.Service, err error) {
 		t.server.HTTP.Handle("/ruler/ring", t.ruler)
 		cortex_ruler.RegisterRulerServer(t.server.GRPC, t.ruler)
 
+		// Prometheus Rule API Routes
+		t.server.HTTP.Path("/prometheus/api/v1/rules").Methods("GET").Handler(t.httpAuthMiddleware.Wrap(http.HandlerFunc(t.ruler.PrometheusRules)))
+		t.server.HTTP.Path("/prometheus/api/v1/alerts").Methods("GET").Handler(t.httpAuthMiddleware.Wrap(http.HandlerFunc(t.ruler.PrometheusAlerts)))
```

Comment:
Do these need to also support POST? I know we saw some issues around query where some clients actually made a POST request for this data which is strange.

Reply:
Cortex only exposes `GET` and the Prom docs only mention `GET` as well: https://prometheus.io/docs/prometheus/latest/querying/api/#alerts